### PR TITLE
only install perl on el6

### PR DESCRIPTION
**Pull request by sean797**

### CODE REVIEW

File: tasks/main.yaml [hunk 1]
```diff
@@ -8,39 +8,42 @@
 - name: Make sure Perl is installed
   package:
     name: perl
+  when:
```
Comment (jtyr) [inline tasks/main.yaml]: Please use the condition formatting like elsewhere in the original file.

File: tasks/main.yaml [hunk 2]
```diff
@@ -8,39 +8,42 @@
...
 
 - name: Get content of the Grub config file for RedHat <= 6
   shell: cat "{{ grub_cmdline_config_v1 }}" | perl -ane '/(?=.*{{ grub_cmdline_add_args | join(")(?=.*") }})/ && print "YES"'
   register: grub_content
   changed_when: grub_content.stdout | length == 0
-  when: >
-    ansible_os_family == "RedHat" and
-    ansible_distribution_major_version | int <= 6
+  when:
+    - ansible_os_family == "RedHat"
```
Comment (jtyr) [inline tasks/main.yaml]: Please do not modify formatting of the conditions.

**Review by jtyr — CHANGES_REQUESTED** [review]

File: tasks/main.yaml (hunk 2 above)
Comment (sean797) [reply tasks/main.yaml]: Using the logical ‘and’ is a lot easier to read and for that reason is generally considered best practice. But I can change it back if required?

Comment (sean797) [reply tasks/main.yaml]: Also the `ansible_os_family == "RedHat"` is moot due to:

https://github.com/jtyr/ansible-grub_cmdline/blob/2bfaef2db63ca2618cf9f456d02f59ca0e1e69aa/tasks/main.yaml#L3-L6

Are you okay with removing that?

Comment (jtyr) [reply tasks/main.yaml]: I prefer if the `and` is explicitly written there. So please change it back.
Yes, you can remove the OS check if it's checked on the top of the file.

**Comment by sean797** [comment]
Updated. thanks @jtyr

**Comment by jtyr** [comment]
Thanks for your contribution, @sean797!

**Pull request merged**